tools/failures.py

```diff
@@ -113,19 +113,28 @@ def format_in_table(active_jobs, master):
     print "Total jobs %s" % (sum_removed + sum_remaining)
 
 
-def insert_in_database(to_remove, date=None):
+def insert_in_database(to_insert, date=None):
     if not date:
         date = datetime.datetime.now().strftime('%Y-%m-%d')
     else:
         date = date.strftime('%Y-%m-%d')
 
     run_query('delete from seta where date="%s"' % date)
-    for jobtype in to_remove:
+    for jobtype in to_insert:
         query = 'insert into seta (date, jobtype) values ("%s", ' % date
         query += '"%s")' % jobtype
         run_query(query)
 
 
+def prepare_the_database():
+    # wipe up the job data older than 90 days
+    date = datetime.datetime.now() - datetime.timedelta(days=90)
+    outdate_jobs = run_query("select jobtype from seta where date>'%s'" % date)
+    if len(outdate_jobs) > 0:
+        print "we have %s outdate jobs need to remove" % len(outdate_jobs)
+    run_query("delete from seta where date>='%s'" % date.strftime('%Y-%m-%d'))
```

Comment:
please verify the query is correct, I think we want less than, not greater than:
"delete from seta where date<='%s'"